### DISCUSSION TITLE
Faster stabilization for Max Lens Mod 2.0

### DISCUSSION THREAD

**Discussion by markusfinholt** [gopro/labs#648] [Feature Requests]
The Max Stabilization on the hero 12 lags behind on rotational movement. So much that it makes you dizzy. Is it possible to create a stabilization preset that stabilizes the image less, so it reacts faster to movement?
Example: https://youtu.be/vw1tN0WcaS8?si=Tj4Q43sDkSI3t-OY&t=202
Check out the video at 03:23, it shows how long it takes for the stabilization to catch up.

**Comment by dnewman-gpsw** [comment]
If Labs can working out how to reduce HS, it will, already on the list.
